### PR TITLE
added mesut's code

### PR DESCRIPTION
**Pull request by henryygu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+.idea/inspectionProfiles/profiles_settings.xml
```
Comment (Pablo-source) [inline .gitignore:2]: Note: please check this line

File: Automation/02 Clean_downloads_folder_mesut.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import os
+import shutil
```
Comment (Pablo-source) [inline Automation/02 Clean_downloads_folder_mesut.py]: Please remove this import because it has been used

**Review by Pablo-source — CHANGES_REQUESTED** [review]
I would approve this only when you remove that import library

File: Automation/02 Clean_downloads_folder_mesut.py (hunk 2 above)
Comment (henryygu) [reply Automation/02 Clean_downloads_folder_mesut.py]: I've removed Shutil import

Comment (Pablo-source) [reply Automation/02 Clean_downloads_folder_mesut.py]: I am happy with these changes.

**Pull request merged**